parallel_bitaxe_benchmark.py

```
@@ -0,0 +1,517 @@
+import requests
+import time
+import json
+import signal
+import sys
+import argparse
+import threading
```

Ruff (0.11.9)

`threading` imported but unused

Remove unused import: `threading`

(F401)